storage/local/chunk/varbit.go

```diff
@@ -322,6 +322,11 @@ func (c varbitChunk) UnmarshalFromBuf(buf []byte) error {
 // encoding implements chunk.
 func (c varbitChunk) Encoding() Encoding { return Varbit }
 
+// Utilization implements chunk.
+func (c varbitChunk) Utilization() float64 {
+	return float64(c.nextSampleOffset()) / float64(cap(c))
```

Comment:
You might want to count the footer part at the end of the chunk as used, too. Not sure about the exact math, but with the current implementation, your utilization can never reach 100% (which might or might not be what you want).